lib/auth-providers/cmd.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
 
       const output = spawnSync(cmd, args, {
         env: Object.assign({}, process.env, cmdEnv),
-        windowsHide: true
+        windowsHide: true,
+        shell: true
```

Comment:
Can we only do this for Windows machines? I'd hate to change this for other platforms if not needed